src/js/controller/MutationDetailsController.js

```diff
@@ -324,6 +324,17 @@ function MutationDetailsController(
 			new Mutation3dController(mutationDetailsView, mainView,
 				_mut3dVisView, view3d, pdbProxy, mutationUtil, gene);
 		}
+
```

Comment:
It might be better to move this code segment into a view component, preferably into the corresponding view where we actually initialize the table. This may not be safe, since component initialization might be asynchronous depending on how we retrieve the data.